src/iocore/net/ConnectionTracker.cc

```diff
@@ -222,25 +287,46 @@ ConnectionTracker::Group::should_alert(std::time_t *lat)
   return zret;
 }
 
+void
+ConnectionTracker::Group::release()
+{
+  if (_count >= 0) {
```

Comment:
Shouldn't that be `> 0`? Was it wrong in the original?

Reply:
Yes, this was in the original. I wondered about this being wrong too.